README.md

```diff
@@ -26,8 +26,16 @@ This implies that the charm needs to detect existing and new service accounts in
 
 ```shell
 juju add-model <my-model>
-juju deploy s3-integrator --channel latest/edge
-juju deploy spark-integration-hub-k8s --channel edge
+juju deploy spark-integration-hub-k8s --channel edge --trust
+```
+
+You can use the `spark-integration-hub` to automatically configure Spark service account. For instance, you can inject s3 credentials by deploying the `s3-integrator` and then relating with the `spark-integration-hub` 
+
+```shell
+juju deploy s3-integrator --channel 1/edge \
+  --config path=<path> \
+  --config endpoint=<s3-endpoint> \
+  --config bucket=<bucket>
 juju relate spark-integration-hub-k8s s3-integrator
```

Comment:
For what it's worth, I think the Juju team would like this to be:

```suggestion
juju integrate spark-integration-hub-k8s s3-integrator
```

And you also then have three "integrate" words in the same line :smile:.